<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BalanceBubble</class>
 <widget class="QWidget" name="BalanceBubble">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>218</width>
    <height>178</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>218</width>
    <height>178</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>178</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#BalanceBubble{
background-color:transparent
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>160</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>160</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>10</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="lblFirst">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Transparent</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="textTransparent">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>0.00 pivx</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblSecond">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Shielded</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="textShielded">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>0.00 pivx</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
